mod_ngarn/__init__.py

```diff
@@ -36,12 +36,14 @@ def script():
 )
 def run(queue_table, limit, max_delay):
     """Run mod-ngarn job"""
-    table_name = utils.sql_table_name(queue_table)
+    queue_table_schema, queue_table_name = queue_table.splt(".")
```

Comment:
should not just split `.` because user not always put in the schema, should provide a default schema